_posts/2019-04-12-rails-girls-buenos-aires-2019.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title:  "Rails Girls: growing as a community"
+date: 2019-04-12 10:00:00
+categories: ["learning", "rails"]
+author: "luciano"
+---
+
+In a [previous article](https://www.ombulabs.com/blog/learning/rails/another-successful-rails-girls-event.html) we talked about one of the [Rails Girls](http://railsgirls.com) events that we did in 2018. Fast forward one year to our first event of 2019 at [A3](https://areatresworkplace.com), a new coworking space located in Buenos Aires.
```

Comment:
It might sound off to some people:

```
events that we did in 2018.
```

Maybe change to:

```
events that we sponsored and co-organized in 2018.
```

Also: 

```
Fast forward one year to our first event of 2019 at [A3](https://areatresworkplace.com), a new coworking space located in Buenos Aires.
```

This sentence doesn't make much sense. It's like an intro to a sentence that doesn't happen. It could be something like this: 

```
Fast forward one year to our first event of 2019 at [A3](https://areatresworkplace.com), a new coworking space located in Buenos Aires, we were once again sponsors and volunteers at another successful Rails Girls event. 
```